src/OnboardingSPA/pages/Steps/BasicInfo/basicInfoForm.js

```diff
@@ -129,94 +129,106 @@ const BasicInfoForm = () => {
 	}, [ debouncedFlowData ] );
 
 	return (
-		<div className="basic-info">
-			<div className={ `${ isError ? 'error__show' : 'error__hide' }` }>
-				{ __( content.error.title, 'wp-module-onboarding' ) }
-			</div>
-			<div className="basic-info-form">
-				<div className="basic-info-form__left">
-					<TextInput
-						title={ sprintf(
-							__(
-								content.siteTitle.title,
-								'wp-module-onboarding'
-							),
-							translations( 'Site' )
-						) }
-						hint={ __(
-							content.siteTitle.hint,
-							'wp-module-onboarding'
-						) }
-						placeholder={ sprintf(
-							__(
-								content.siteTitle.placeholder,
+		<Animate
+			type={"fade-in"}
+			after={
+				typeof flowData === 'object' && typeof socialData === 'object'
```

Comment:
Can we use Double Quotes as I saw in the docs usage.
<img width="784" alt="Screenshot 2023-01-16 at 3 41 20 PM" src="https://user-images.githubusercontent.com/48691514/212653169-976c9780-6ef6-465f-990a-5415dac87979.png">